projects/plugins/jetpack/_inc/client/mixins/emitter/index.js

```diff
@@ -1,12 +1,3 @@
-// Here we make sure we’re not using Node’s core events module,
-// because it contains a `domain` property and it clashes with our
-// `Site` module that also has the same property.
-//
-// By adding a slash we force `require` to go into directory mode and to
-// load the module from `node_modules/` instead of the core’s `events.js`
-// file. Webpack uses the same module on the client side, too, which
-// makes for a nice consistency.
```

Comment:
#11677 changed the code here but didn't remove this comment that it apparently made obsolete.